src/go/k8s/apis/redpanda/v1alpha1/cluster_types.go

```diff
@@ -169,21 +164,35 @@ type ClusterList struct {
 
 // RedpandaConfig is the definition of the main configuration
 type RedpandaConfig struct {
-	RPCServer     SocketAddress `json:"rpcServer,omitempty"`
-	KafkaAPI      SocketAddress `json:"kafkaApi,omitempty"`
-	AdminAPI      SocketAddress `json:"admin,omitempty"`
-	DeveloperMode bool          `json:"developerMode,omitempty"`
-	TLS           TLSConfig     `json:"tls,omitempty"`
+	RPCServer     SocketAddress      `json:"rpcServer,omitempty"`
+	KafkaAPI      []KafkaAPIListener `json:"kafkaApi,omitempty"`
+	AdminAPI      []AdminAPI         `json:"adminApi,omitempty"`
```

Comment:
I think I like `AdminAPI` better than `KafkaAPIListener`, but consistency would be nice too.  We'll have proxy too at some point:

```
[]KafkaAPIListener
[]AdminAPI
[]PandaproxyAPI
```

Not too important, just pointing it out.

Reply:
yeah, good point. let's rename KafkaAPIListener to KafkaAPI. I'll create a PR